New Post MiXiT 2022 (#357)
BedrockStreaming/tech.bedrockstreaming.com: _posts/2022-07-28-retour-sur-mixit-2022.md

---
layout: post
title: Retour sur la conférence MiXiT 2022
description: Plusieurs Bedrockiens étaient présents à MiXiT 2022, voici un résumé de leurs conférences préférées.
author: [s_haim,e_perrin,j_mastounga,s_zoccarato]
tags: [conference, agile]
color: rgb(251,87,66)
thumbnail: "/images/posts/mixit-crepes.jpg"
---
!["MiXiT, la conférence avec des crêpes et du cœur"](/images/posts/mixit-crepes.jpg)

[MiXiT](https://mixitconf.org/) est une conférence "avec des crêpes et du cœur" qui se déroule à Lyon. Les sujets sont assez variés abordant autant l'agilité, que la programmation, le droit ou encore l'histoire de l'informatique.

Voici un résumé des conférences de l'édition 2022 qui nous ont le plus marquées.

## How to build the alert system that France deserves?

> par Gaël Musquet

Reviewer: Je ne suis pas convaincu de l'utilité de ce "par XYZ" (sur cette conf et sur d'autres). Inclure le nom dans le texte juste après est plus "élégant". D'ailleurs, c'est ce que tu as fait ;-)

* [Page du talk sur le site de MiXiT et voir le replay](https://mixitconf.org/2022/meet-null-the-unknown)

## Parlez de vous, faites des feedbacks

par Julie Quillé

Le feedback est un outil communicationnel qui permet de formuler un avis sur une situation passée dans le but de gérer les situations futures.

On peut trouver plusieurs formes de feedbacks :
le feedback est à destination de la personne, pour l’aider à s’améliorer. Elle peut décider de le suivre ou non,
la demande que l’on fait à quelqu’un est à notre bénéfice (on demande à la personne de changer un comportement qui nous gêne) en laissant la possibilité à la personne de décider si elle veut ou non répondre favorablement à cette demande,
l’exigence qui est aussi à notre bénéfice, mais pour laquelle on ne laisse pas le choix (dans le cadre d’une relation hiérarchique)

Julie propose un modèle de feedbacks basé sur la CNV (Communication Non Violente) et qui peut se résumer de la manière suivante :

1- On vérifie la disponibilité de la personne en lui demandant si elle est d’accord pour qu’on lui fasse des feedbacks et sous quelle forme.

2- On formule le feedback :

> Décrire une Observation, les faits (= pas de jugement)

Reviewer: ceci n'est pas formaté correctement : ça remonte tout sur une ligne

* les barrières infinies aka les bonnes excuses pour ne pas passer à l’action : j’aimerais, mais … / je pourrais, mais …
* les sensations de déjà vu : les blocages et les situations récurrentes

Une fois qu’on a repéré une de ses croyances limitantes, on l’écrit sur une feuille et on va ensuite décomposer cette croyance et réfléchir à :
* son origine : d’où nous vient cette croyance ? depuis combien de temps fait-elle partie de nous ? nous vient-elle de notre éducation ?
* les bénéfices : quels bénéfices nous apporte cette croyance ? qu’est-ce qu’elle nous permet ?
* les inconvénients / les freins : en quoi cette croyance nous gêne et quels sont les impacts sur notre vie (pro ou perso) ?
* les contradictions : a-t-on déjà fait quelque chose ou été dans une situation qui vient contredire cette croyance ?

On va ensuite venir agrémenter notre croyance avec tous ces éléments puis, pour finir, transformer notre croyance limitante en une croyance aidante.

[Page du talk sur le site de MiXiT et voir le replay](https://mixitconf.org/2022/-arretez-l-auto-sabotage-et-sortez-de-votre-boucle-systemique-)

> ## Comment fonctionne un gestionnaire de mot de passe

Reviewer: ```suggestion
## Comment fonctionne un gestionnaire de mots de passe
```

Les mots de passe sont partout. Ils nous permettent d'accéder à nos photos, nos comptes bancaires, nos documents de santé et bien d'autres données sensibles que l'on ne souhaite pas voir aux mains d'individus que l’on ne connaît pas.

> Tout le monde sait que l'on doit avoir des longs mots de passe mais comment tous les retenir ? C'est là que les gestionnaires de mot de passe entrent en jeu. Mais peut-on leur faire confiance ? Comment ça marche au juste ? C'est à cette question qu'à souhaiter répondre Eric Daspet pendant sa conférence.

Reviewer: ```suggestion
Tout le monde sait que l'on doit avoir des longs mots de passe mais comment tous les retenir ? C'est là que les gestionnaires de mot de passe entrent en jeu. Mais peut-on leur faire confiance ? Comment ça marche au juste ? C'est à cette question qu'a souhaité répondre Eric Daspet pendant sa conférence.
```

> Le rôle d'un gestionnaire de mot de passe est de permettre à son utilisateur d'utiliser qu'un seul mot de passe pour ensuite laisser l'outil générer et mémoriser tous les autres mots de passe. On a plus qu'à retenir un seul mot de passe et par conséquent ce dernier peut-être long et complexe, l'exercice de mémoire sera moins compliqué que si on en avait plusieurs à retenir.

Reviewer: ```suggestion
Le rôle d'un gestionnaire de mots de passe est de permettre à son utilisateur d'utiliser qu'un seul mot de passe pour ensuite laisser l'outil générer et mémoriser tous les autres mots de passe. On a plus qu'à retenir un seul mot de passe qui peut donc être long et complexe. L'exercice de mémoire sera alors moins compliqué que si on en avait plusieurs à retenir.
```

À travers son exposé, on découvre un peu plus tous les procédés de cryptographie utilisés afin de gérer les mots de passe que l'on va créer ou modifier en utilisant ces outils.
Grâce à de nombreux schémas, il explique clairement les différentes étapes de chiffrements utilisées que ce soit pour la création du mot de passe maître, la création et le changement des mots de passe, l'affichage des mots de passe et même le fonctionnement du partage de mots de passe (lorsque celui-ci existe dans l'outil).

> On découvre pendant cette heure que les gestionnaires de mot de passe ne cherchent pas à réinventer la roue en matière de cryptographie mais s'appuient sur des concepts déjà éprouvés et robustes. On apprend aussi que tout est chiffré de bout en bout et que seul celui qui détient le mot de passe maître (l'utilisateur donc, même l'outil ne le connaît pas et n'en a pas besoin) peut interagir avec les mots de passe créer. Rassurant non ? En tout cas, me voilà maintenant prêt à expliquer autour de moi pourquoi il est grand temps de passer à un gestionnaire de mot de passe !

Reviewer: ```suggestion
On découvre pendant cette heure que les gestionnaires de mots de passe ne cherchent pas à réinventer la roue en matière de cryptographie mais s'appuient sur des concepts déjà éprouvés et robustes. On apprend aussi que tout est chiffré de bout en bout et que seul celui qui détient le mot de passe maître (l'utilisateur donc, même l'outil ne le connaît pas et n'en a pas besoin) peut interagir avec les mots de passe créés. Rassurant, non ? En tout cas, me voilà maintenant prêt à expliquer autour de moi pourquoi il est grand temps de passer à un gestionnaire de mot de passe !
```

[Page du talk sur le site de MiXiT et voir le replay](https://mixitconf.org/2022/comment-fonctionne-un-gestionnaire-de-mots-de-passe-)

## Optimiser votre revue de code avec le rebase interactif

GIT est un outil bien connu des développeurs de nos jours, mais dès lors que l'on s'écarte des commandes traditionnelles (checkout, commit et push), on sait bien moins ce que l'on peut faire d'autre avec.

Sonia Seddiki nous explique ici comment rendre la revue de code, souvent longue et fastidieuse, plus simple et agréable pour nos collègues avec quelques astuces qu'elle a partagées avec nous lors d'un live coding.

> Contrairement à l'idée que j'en avais, le rebase interactif n'est pas là que pour nettoyer les noms de commit sans aucun sens que j'avais mis dans la précipitation mais que c'était un outil bien plus puissant.

Reviewer: ```suggestion
Contrairement à l'idée que j'en avais, le rebase interactif n'est pas là que pour nettoyer les noms de commit sans aucun sens que j'avais mis dans la précipitation mais c'est un outil bien plus puissant.
```

Il s’agit d’une conférence passionnante, dérangeante et éclairante que je conseille à tous. Le début piétine un peu, mais le voyage en vaut la peine.

[Page du talk sur le site de MiXiT et voir le replay](https://mixitconf.org/2022/violence-hermeneutique-comment-eviter-le-malaise-)

## Designer pour le service public

Cela peut faire un peu peur dit comme ça, mais je suis allé sceptique à cette conférence d’Anne-Sophie Tranchet. J’étais rattaché à une image peu flatteuse des outils du service public, alors que ces derniers ont connu une vraie progression ces dernières années. Anne-Sophie fait partie du programme beta.gouv qui intervient auprès des administrations pour les services numériques.

> C’est armé des bonnes pratiques de nos métiers que Beta.gouv a la mission de transformer et d’accompagner les services publics. On y apprendra le parcours d’Anne-Sophie, ce que cela veut dire de travailler pour le service public ainsi que les projets et challenges qui en découlent. Leur méthodologie centrée utilisateur leur permet de travailler en itération, et de délivrer de la valeur, en incubation d’abord, puis jusqu’à un développement national en fonction des retours sur le service.

Reviewer: ```suggestion
C’est armé des bonnes pratiques de nos métiers que Beta.gouv a la mission de transformer et d’accompagner les services publics. On y apprendra le parcours d’Anne-Sophie, ce que travailler pour le service public veut dire, ainsi que les projets et challenges qui en découlent. Leur méthodologie centrée utilisateur leur permet de travailler en itération, et de délivrer de la valeur, en incubation d’abord, puis jusqu’à un développement national en fonction des retours sur le service.
```

On peut citer quelques réalisations Beta.gouv comme la plateforme dossierfacile.com, qui facilite la création de dossier pour une location, ou 1000 Premiers Jours, qui délivre des informations et un accompagnement sur la grossesse et les 2 premières années de l’enfant

[Page du talk sur le site de MiXiT et voir le replay](https://mixitconf.org/2022/designer-pour-le-service-public)

> ### Mes autres conférences coup de coeur

Reviewer: "mes" ou "nos" ?
